style.css

```diff
@@ -0,0 +1,786 @@
+    width: 100%;
+    border-radius: 50%;
+  }
+
+  .marshmello {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .marshmello img {
+    width: 100%;
+    border-radius: 45%;
+  }
+
+  .ice {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .ice img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .eminem {
+    margin-top: 15px;
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .eminem img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .speaker-text {
+    width: 50%;
+  }
+
+  .speaker-text h1 {
+    font-family: "LatoWebBlack", sans-serif;
+  }
+
+  .job {
+    font-family: "LatoWeb", sans-serif;
+    font-style: italic;
+    color: #ec5242;
+  }
+
+  .small-grey-line {
+    width: 40px;
+    height: 2px;
+    background: #9e9e9e63;
+  }
+
+  .description {
+    font-family: "LatoWebThin", sans-serif;
+    font-size: medium;
+    font-weight: bold;
+  }
+
+  .button-artists {
+    display: flex;
+    background-color: white;
+    margin-top: 20px;
+    width: 90%;
+    height: 50px;
+    margin-bottom: 20px;
+    border-style: none;
+    border: 2px solid #9e9e9e63;
+    font-family: "LatoWebBold", sans-serif;
+    font-size: medium;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .button-artists img {
+    width: 25px;
+    padding-bottom: 4px;
+  }
+
+  .upperpart {
+    display: none;
+  }
+
+  .header-p-desktop {
+    display: none;
+  }
+
+  .header-day-desktop {
+    display: none;
+  }
+
+  .header-place-desktop {
+    display: none;
+  }
+
+  .footer-desktop {
+    display: none;
+  }
+
+  .link-desktop {
+    display: none;
+  }
+
+  .grid-container-desktop {
+    display: none;
+  }
+
+  .rights-desktop {
+    display: none;
+  }
+
+  .logo-desktop {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #stickers {
+    display: none;
+  }
+
+  .upperpart {
+    display: flex;
+    flex-direction: row;
+    background-color: #272a31;
+    height: min-content;
+    align-items: center;
+    place-content: flex-end;
+    padding-right: 15%;
+  }
+
+  .upperpart img {
+    width: 30px;
+    padding-right: 20px;
+  }
+
+  .upperpart h4 {
+    color: white;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: small;
+    margin-right: 25px;
+  }
+
+  .artists {
+    display: block;
+  }
+
+  .artists h2 {
+    text-align: center;
+  }
+
+  .container-speakers {
+    display: flex;
+  }
+
+  .speaker {
+    width: 50vw;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .speaker-text {
+    width: 50%;
+  }
+
+  .close-hamburger {
+    display: none;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .background-image {
+    position: relative;
+    height: 750px;
+  }
+
+  .header-h1 {
+    color: #d72311;
+    padding-left: 15%;
+    font-family: "LatoWebBlack", sans-serif;
+    font-size: xxx-large;
+  }
```

Comment:
- [ ] Please make sure to stick to the given Behance design as I noticed you're not using the appropriate font family which should be "Cocogoose" for the headings and "Lato" for the paragraphs. As a frontend developer you have to pay attention to small details like this as they give your UI a more polished look and feel. ✔